maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/controller/FinancialAssessmentController.java

```diff
@@ -0,0 +1,107 @@
+package gov.uk.courtdata.assessment.controller;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import gov.uk.courtdata.assessment.service.FinancialAssessmentService;
+import gov.uk.courtdata.assessment.validator.FinancialAssessmentValidationProcessor;
+import gov.uk.courtdata.dto.ErrorDTO;
+import gov.uk.courtdata.dto.FinancialAssessmentDTO;
+import gov.uk.courtdata.model.assessment.CreateFinancialAssessment;
+import gov.uk.courtdata.model.assessment.UpdateFinancialAssessment;
+import gov.uk.courtdata.util.DateUtil;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.annotation.PostConstruct;
+import java.time.LocalDateTime;
+
+@RestController
+@RequestMapping("/assessment")
```

Comment:
This is a generic name, as there will be other sorts of assessments. I'd propose, this needs to be more specific with what kind of assessment.